### PR TITLE
minor: typing for ReprFailDoctest

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by blueyed** [comment]
Job canceled manually.

File: src/_pytest/doctest.py [hunk 1]
```diff
@@ -113,11 +115,10 @@ def _is_doctest(config, path, parent):
 
 
 class ReprFailDoctest(TerminalRepr):
-    def __init__(self, reprlocation_lines):
-        # List of (reprlocation, lines) tuples
+    def __init__(self, reprlocation_lines: List[Tuple[ReprFileLocation, List[str]]]):
```
Comment (bluetech) [inline src/_pytest/doctest.py]: Can the Lists be Sequences?

Comment (blueyed) [reply src/_pytest/doctest.py]: Probably.. just based it on the comment.. :D - this was just from a stash when working at that code.
Should I change it?

Comment (bluetech) [reply src/_pytest/doctest.py]: Yes, it doesn't modify the input so Sequence is better all around.

LGTM otherwise.

Comment (blueyed) [reply src/_pytest/doctest.py]: Changed both `List`s now.  Hope that's ok? :)

**Pull request merged**